packages/public-types-lib/src/index.ts

```diff
@@ -117,7 +117,7 @@ export interface ISettlementBatchTransfer {
 	amount: string;
 	batchId: string;
 	batchName: string;
-	journalEntryId: string;
+	journalEntryId: string | null;
```

Comment:
Just to confirm, this is due to the initial transfer not having the `journalEntryId`, it will only be assigned later once A&B has been invoked?
Maybe we should add a comment here explaining that?